examples/accounts/README.md

```diff
@@ -2,7 +2,7 @@
 
 This example demonstrates the capabilities of the `pkg/accounts` packages through a web-based interface.
 
-## Features
+## What it demonstrates
```

Comment:
Hmm, `Features` makes more sense to me 🤔

Reply:
hmm not sure... It's not like a real app with features, it's a brief demo that demonstrates features from a library.